core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -71,44 +99,176 @@ describe('About Page', () => {
+  });
+
+  it('should set screen type to tablet when window width is between 362 and 768', () => {
+    spyOn(windowDimensionsService, 'getWidth').and.returnValue(500);
+    component.setScreenType();
+    expect(component.screenType).toEqual('tablet');
+  });
+
+  it('should set screen type to desktop when window width is greater than 768', () => {
+    spyOn(windowDimensionsService, 'getWidth').and.returnValue(800);
+    component.setScreenType();
+    expect(component.screenType).toEqual('desktop');
+  });
+
+  it('should set showNavigationArrowsForCarousel to "false" if window width is greater than 640', () => {
+    spyOn(windowDimensionsService, 'getWidth').and.returnValue(641);
+    component.setScreenType();
+    expect(component.showNavigationArrowsForCarousel).toBeFalse();
+  });
+
+  it('should set showNavigationArrowsForCarousel to "true" if window width is lesser than 641', () => {
+    spyOn(windowDimensionsService, 'getWidth').and.returnValue(640);
+    component.setScreenType();
+    expect(component.showNavigationArrowsForCarousel).toBeTrue();
   });
 
   it(
-    'should redirect guest user to the login page when they click' +
-      'create lesson button',
+    'should obtain new form link whenever the selected' + 'language changes',
     () => {
-      spyOn(
-        siteAnalyticsService,
-        'registerCreateLessonButtonEvent'
-      ).and.callThrough();
-      component.onClickCreateLessonButton();
-
-      expect(
-        siteAnalyticsService.registerCreateLessonButtonEvent
-      ).toHaveBeenCalledWith();
+      component.ngOnInit();
+      spyOn(component, 'setPartnershipsFormLink');
+      translateService.onLangChange.emit();
+
+      expect(component.setPartnershipsFormLink).toHaveBeenCalled();
     }
   );
 
-  it('should register correct event on calling onClickVisitClassroomButton', () => {
-    spyOn(
-      siteAnalyticsService,
-      'registerClickVisitClassroomButtonEvent'
-    ).and.callThrough();
-    component.onClickVisitClassroomButton();
+  it('should set the correct form link for English language', () => {
+    translateService.currentLang = 'en';
+    const formLink = 'https://forms.gle/Y71U8FdhQwZpicJj8';
+    component.setPartnershipsFormLink();
 
-    expect(
-      siteAnalyticsService.registerClickVisitClassroomButtonEvent
-    ).toHaveBeenCalledWith();
+    expect(component.partnershipsFormLink).toBe(formLink);
+  });
+
+  it('should set the correct form link for Portuguese language', () => {
+    translateService.currentLang = 'pt-br';
+    const formLink =
+      'https://docs-google-com.translate.goog/forms/d/e/1FAIpQLSdL5mjFO7RxDtg8yfXluEtciYj8WnAqTL9fZWnwPgOqXV-9lg/viewform?_x_tr_sl=en&_x_tr_tl=pt&_x_tr_hl=en-US&_x_tr_pto=wapp';
+    component.setPartnershipsFormLink();
+
+    expect(component.partnershipsFormLink).toBe(formLink);
+  });
+
+  it('should set the correct form link for general languages', () => {
+    translateService.currentLang = 'fr';
+    const formLink =
```

Comment:
Maybe have this as a global constant somewhere. This makes it easier to maintain this test and the corresponding function in .ts as we'll only have to change this link in one place instead of two.

Reply:
I don't see it changes, it's still a full link.